tests/test_vcs_git/test_vcs_git.py

```diff
@@ -0,0 +1,42 @@
+from os.path import dirname, join
+from shutil import copyfile
+
+from dxr.testing import DxrInstanceTestCase
+
+from nose import SkipTest
+from nose.tools import ok_
+
+
+class GitTests(DxrInstanceTestCase):
+    """Tests that the diff links for files go somewhere helpful"""
```

Comment:
From this [build log](https://ci.mozilla.org/job/dxr-github/931/console), it looks like all these tests are failing.
However I'm not able to reproduce the failures. Any idea what's going wrong?